src/common/SConscript.test

```diff
@@ -0,0 +1,86 @@
+"""Build common libraries"""
```

Comment:
Why the odd filename here?  It'll probably bypass all the python checks because of the filename so they should be updated.

At least these two lines will want to be updated if you want to have SConscript files not named SConscript.
https://github.com/daos-stack/daos/blob/master/.flake8-scons#L14
https://github.com/daos-stack/daos/blob/master/utils/cq/daos_pylint.py#L188

Reply:
I think the right way to fix this is in a subsequent patch to move the build of the test libraries to a different directory so we can use SConscript as the name.

I would do that here but this patch is already large.